app/api/V2/views/parcelviews.py

```diff
@@ -21,31 +21,49 @@ def post(self):
        
         query = "INSERT INTO orders (destination, origin, price, weight, user_id) VALUES (%s,%s,%s,%s,%s)"
         
-        curobject = connection.cursor()
-        curobject.execute(query, (data['destination'],data['origin'],data['weight']*20,data['weight'],current_userid))
+        cursor_object = connection.cursor()
+        cursor_object.execute(query, (data['destination'],data['origin'],data['weight']*20,data['weight'],current_userid))
         connection.commit()
-        return {"message":"parcel craeted, waiting approval"}, 201
+        return {"message":"parcel created, waiting approval"}, 201
 
 
+class GetUserParcels(Resource):
+    """Class to get parcels by a specific user."""
+
+    def get(self, user_id):
+        """Get all parcel by a unique user ID."""
+        current_orders=[]
+        cursor_object = connection.cursor()
+        query = """SELECT * FROM orders WHERE user_id= %s"""
+        cursor_object.execute(query, (user_id,))
+        rows = cursor_object.fetchall()
+        if len(rows)==0:
```

Comment:
You should leave a line space before the `if`, `for loop` blocks and return statement. This will your code more readable.